V1/DTOs/Wallet/WalletDetailRequest.cs

```diff
@@ -0,0 +1,18 @@
+﻿using CoviIDApiCore.V1.DTOs.Credentials;
+using System.ComponentModel.DataAnnotations;
+
+namespace CoviIDApiCore.V1.DTOs.Wallet
+{
+    public class WalletDetailRequest
+    {
+        [StringLength(50)]
+        public string FirstName { get; set; }
+        [StringLength(50)]
+        public string LastName { get; set; }
+        public string Photo { get; set; }
+        public string MobileNumber { get; set; }
```

Comment:
I'm not sure, because this isn't used anywhere yet, but I don't think MobileNumber should be here.

As far as I know, it's not one of the things we return to the verifier.